include/seqan3/range/container/bitcompressed_vector.hpp

```diff
@@ -32,9 +32,9 @@ namespace seqan3
 {
 
 /*!\brief A space-optimised version of std::vector that compresses multiple letters into a single byte.
- * \tparam alphabet_type The value type of the container, must satisfy seqan3::Alphabet and not be `&`.
- * \implements seqan3::ReservableContainer
- * \implements seqan3::Cerealisable
+ * \tparam alphabet_type The value type of the container, must satisfy seqan3::alphabet and not be `&`.
+ * \implements seqan3::reservable_container
```

Comment:
I'm not sure if this matters so much... But reversible is spelled with an I not an A.

Reply:
Good catch! One of the few things I could never wrap my head around... is there a rule?

Reply:
I just googled it... There isn't really a catch all rule haha. I think it's just based on feeling?